source/common/init/BUILD

```diff
@@ -22,6 +22,9 @@ envoy_cc_library(
     name = "target_lib",
     srcs = ["target_impl.cc"],
     hdrs = ["target_impl.h"],
+    external_deps = [
+        "abseil_memory",
+    ],
```

Comment:
6 occurrences. I like the simplicity of WrapUnique. I can seek to mark abseil_memory as a common deps in targets so that I don't have to introduce this deps.
And eventually we can replace WrapUnique along with better c++ stdlib